upstart_msh/cob-stop

```diff
@@ -52,6 +63,9 @@ else
   echo "......(cob-start not running)....."
 fi
 
+#cleaning up roscore
+su $USER -l -c "rosnode cleanup"
```

Comment:
this throws an error: `-su: rosnode: command not found`

can be fixed by 
```
su $USER -l -c "source /opt/ros/indigo/setup.bash && rosnode cleanup"
```

probably replace `indigo` by `mydistro`